src/ui/components/AccordionItem.stories.tsx

```diff
@@ -25,3 +33,24 @@ Open.args = {
   children: <Typo.Body>{children}</Typo.Body>,
   defaultOpen: true,
 }
+
+export const WithActiveSwitch = Template.bind({})
+WithActiveSwitch.args = {
+  title: 'Accordion item with switch',
+  children: <Typo.Body>{children}</Typo.Body>,
+  switchProps: {
+    active: true,
+    toggle: () => null,
```

Comment:
je suggère d'utiliser `action` de storybook, pour que ça log dans storybook lorsqu'on clic dessus et qu'on puisse vérifier que le binding est bien fait

```suggestion
    toggle: action('toggle'),
```